lib/taskwarrior-web/services/builder/base.rb

```diff
@@ -6,13 +6,14 @@ module TaskwarriorWeb::CommandBuilder::Base
     :add => 'add',
     :update => TaskwarriorWeb::Config.version.major >= 2 ? ':id mod' : nil,
     :delete => 'rc.confirmation=no :id delete',
-    :query => TaskwarriorWeb::Config.version > '1.9.2' ? '_query' : 'export',
+    :query => 'export',
```

Comment:
After analysing the taskwarrior code I found that JSON export was added in version 1.9.4. In v2.0.0 (the next release) '_query' was aliased to 'export' and JSON was the default output format. In v2.4.0, the alias was removed and so '_query' was no longer supported.
It seems to me that taskwarrior-web does not support versions of taskwarrior before v1.9.4, and than versions later than 1.9.4 should use 'export' instead of '_query'.
I'll update this functionality in my next commit to use '_query' if the version is 1.9.4.